crates/humanode-runtime/src/lib.rs

```diff
@@ -1597,14 +1599,8 @@ impl_runtime_apis! {
             signature_check: bool,
             select: frame_try_runtime::TryStateSelect
         ) -> Weight {
-            sp_tracing::info!(
-                target: "humanode-runtime",
-                "try-runtime: executing block {:?} / root checks: {:?} / signature check: {:?} / try-state-select: {:?}",
-                block.header.hash(),
-                state_root_check,
-                signature_check,
-                select,
-            );
```

Comment:
Already have it under `try_execute_block` - https://github.com/humanode-network/substrate/blob/56b16fefbd7a5ba07afd94cb37367d078da0ac52/frame/executive/src/lib.rs#L245

Reply:
What's odd is that they use `frame_support::log`; does this mean we got this message kind of printed twice with tracing enabled?

Reply:
Yep, it's printed twice
![image](https://github.com/user-attachments/assets/f8073b80-30c9-4fd7-8120-b9b8786f13a6)

Without - just once, right.